.firefox/manifest.json

```diff
@@ -0,0 +1,17 @@
+{
+  "manifest_version": 2,
+  "name": "SFCC DevTools",
+  "version": "1.0.0",
+  "description": "Support Browser Interaction with VS Code, Eclipse, and SFCC Business Manager via DevTools Web Inspector.",
+  "author": "Red van Workshop",
```

Comment:
A totally Dutch vibe, but not quite a proper name

Reply:
Fixed :)